driver/libcompile.mli

```diff
@@ -0,0 +1,90 @@
+(**************************************************************************)
```

Comment:
I still think the name of this module isn't clear.  What about `Compile_common` or similar?